data/centralized-info/notifications.yml

```
@@ -447,6 +447,45 @@ entries:
   nextSteps: Monitor tunnel health over time and consider deploying [`cloudflared` replicas or load balancers](/cloudflare-one/connections/connect-networks/deploy-tunnels/deploy-cloudflared-replicas/).
   otherFilters: None.
 
+- name: Device connectivity anomaly
+  audience: Zero Trust customers who want to be alerted when the number of devices connected to WARP increases or decreases significantly. For more information, refer to [DEX notifications](/cloudflare-docs/content/cloudflare-one/insights/dex/notifications/).
```

Nice 🙌  Same nit comment as above re: verbiage change following review with Alexa.

```suggestion
  audience: Zero Trust customers who want to be notified when Cloudflare detects a spike or drop in devices connected to the WARP client. For more information, refer to [DEX notifications](/cloudflare-docs/content/cloudflare-one/insights/dex/notifications/).
```